doc/classes/ButtonGroup.xml

```diff
@@ -14,6 +15,7 @@
 			<return type="BaseButton[]" />
 			<description>
 				Returns an [Array] of [Button]s who have this as their [ButtonGroup] (see [member BaseButton.button_group]).
+				[b]Note:[/b] There is no equivalent [code]set_buttons()[/code] method, as new buttons are added by setting the [member BaseButton.button_group] property of the button instead. See this class' description for instructions on adding buttons to a group from the editor.
```

Comment:
To be honest I don't even think this note about a theoretical "setter" is necessary. The line above this literally explains where the buttons come from. If it is necessary, there's so many similar getters out there deserving of similar explanations.
```suggestion
				[b]Note:[/b] There is no equivalent method to set the buttons, as new buttons are added by setting the [member BaseButton.button_group] property of the button instead. See this class's description for instructions on adding buttons to a group from the editor.
```
I would suggest omitting it, to be honest, or only mentioning the "class description" part
```suggestion
```